__tests__/app.test.js

```diff
@@ -77,3 +77,44 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("should respond with 200 and a review object with correct review id", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          review: expect.objectContaining({
+            review_id: expect.any(Number),
```

Comment:
You knwo exactly which review is coming back. Make sure you are specific on line 89